### PR TITLE
1st refactor done. demo status: ok

### LINKED ISSUE #7: survey the naming scheme for files and directories and for symbols in scripts.

### PR DESCRIPTION
**Pull request by loteque**
- renamed classes: simplifies identification
- renamed members to match class names
- renamed source files to match class names
closes issue #7

### CODE REVIEW

File: metastore.gd
Comment (jonathandavidlewis) [inline metastore.gd:1]: Can you record this as a file name change instead of a delete / new change?

File: ResourceContainer.gd [hunk 1]
```diff
@@ -20,16 +22,16 @@ signal updated_container(
...
+    exit_status: Transaction.ExitStatus
 )
 
 func get_container():
     return container
 
 func create_container():
 
-    container = ContainerResource.new(
+    container = Store.new(
```
Comment (jonathandavidlewis) [inline ResourceContainer.gd]: Calling this a "store" instead would be more consistent.

A ContainerResource uses a "store" so it has a "store" property.

**Review by jonathandavidlewis — CHANGES_REQUESTED** [review]
I just made 2 suggestions. Otherwise, big improvements! Nice!

File: metastore.gd
Comment (loteque) [reply metastore.gd:1]: I don't know how to do that

File: ResourceContainer.gd (hunk 1 above)
Comment (loteque) [reply ResourceContainer.gd:34]: that makes sense to me. I waffled on that one for a minute and so just left it. But hearing you say this also gives me confidence to make the change.

File: metastore.gd
Comment (jonathandavidlewis) [reply metastore.gd:1]: You're good. I looked at the git history and it's a file move already. The PR as viewed through github is expressing it as a delete/new.

**Pull request merged**